xml/System.Numerics/Complex.xml

```diff
@@ -2292,6 +2292,8 @@ A return value of `false` does not imply that <xref:System.Numerics.INumberBase%
 
 This function returns `true` for a complex number `a + bi` where `b` is zero.
 
+This function also returns `true` for a complex number `a + bi` where `a` is infinity, but not `b`, and does not return true for <xref:System.Numerics.Complex.Infinity>.
```

Comment:
I'm not sure this adds much, as the first comment covers this entirely.

As indicated, the function returns true for any complex number `a + bi` where `b` is zero, that by definition includes `a == infinity`. The requirement that `b` be zero then excludes `b` being anything else and that is also why `Complex.Infinity` returns false, because complex infinity is `inf, inf`

Reply:
Yes, that seems logical.